src/main/webapp/itempagescript.js

```diff
@@ -0,0 +1,23 @@
+function getItemId() {
+  const queryString = window.location.search;
+  console.log(queryString);
+}
+
+async function getItemPageComments() {
+  const response = await fetch("/itempagedata");
+  const comments = await response.json();
```

Comment:
Add a try-catch block for the ```fetch``` and ```reponse.json()``` statements since it is possible that they throw an exception. In the catch block make sure to output the type of error that you found and provide a way to end the function safely.